gcp/gcp.go

```diff
@@ -487,6 +487,93 @@ const (
 	Owner   Role = "OWNER"
 )
 
+// Search calls google.com/cloudprint/search to query printers owned by the user/robot.
```

Comment:
Group this function with the other public functions, maybe place it after Printer().